tests/spawner/conftest.py

```diff
@@ -61,569 +61,384 @@ def random_strategy_spawn_ticks() -> list[int]:
+        647200,
+        650800,
+        652600,
+        656200,
+        659800,
+        662500,
+        665200,
+        670600,
+        673300,
+        676900,
+        679600,
+        683200,
+        685000,
+        688600,
+        691300,
+        694900,
+        697600,
+        701200,
+        704800,
+        709300,
+        711100,
+        714700,
+        717400,
+        719200,
+        722800,
+        725500,
+        728200,
+        731800,
+        735400,
+        738100,
+        741700,
+        745300,
+        748000,
+        751600,
+        753400,
+        756100,
+        758800,
+        760600,
+        765100,
+        768700,
+        771400,
+        775900,
+        778600,
+        781300,
+        784000,
+        786700,
+        789400,
+        795700,
+        800200,
+        803800,
+        806500,
+        810100,
+        812800,
+        815500,
+        820000,
+        824500,
+        827200,
+        830800,
+        833500,
+        837100,
+        839800,
+        843400,
+        847900,
+        850600,
+        853300,
+        857800,
+        860500,
+        862300,
+        866800,
+        870400,
+        872200,
+        875800,
+        879400,
+        882100,
+        886600,
+        890200,
+        893800,
+        896500,
+        899200,
+        902800,
+        907300,
+        910900,
+        915400,
+        919000,
+        921700,
+        925300,
+        928000,
+        930700,
+        934300,
+        937900,
+        940600,
+        943300,
+        946900,
+        949600,
+        953200,
+        956800,
+        959500,
+        962200,
+        965800,
+        968500,
+        970300,
+        974800,
+        978400,
+        982900,
+        985600,
+        988300,
+        991900,
+        995500,
+        997300,
+        1001800,
+        1008100,
+        1010800,
+        1014400,
+        1016200,
+        1019800,
+        1023400,
+        1026100,
+        1028800,
+        1032400,
+        1036000,
+        1039600,
+        1042300,
+        1045000,
+        1047700,
+        1050400,
+        1053100,
+        1055800,
+        1060300,
+        1065700,
+        1069300,
+        1072900,
+        1076500,
+        1081000,
+        1083700,
+        1088200,
+        1090900,
+        1093600,
+        1098100,
+        1100800,
+        1104400,
+        1107100,
+        1110700,
+        1113400,
+        1116100,
+        1118800,
+        1123300,
+        1126900,
+        1128700,
+        1132300,
+        1135000,
+        1137700,
+        1141300,
+        1145800,
+        1149400,
+        1152100,
+        1153900,
+        1158400,
+        1161100,
+        1163800,
+        1167400,
+        1170100,
+        1174600,
+        1177300,
+        1183600,
+        1185400,
+        1189000,
+        1191700,
+        1195300,
+        1198900,
+        1202500,
+        1206100,
+        1208800,
     ]
 
```

Comment:
Why do you use so many more ticks for the demand strategy than for the random strategy?

Reply:
This is historically grown :D
There is no other particular reason than this. If necessary I will decrease the amount of ticks in the future.